src/main-client-v2.ts

```diff
@@ -3362,30 +3685,6 @@ export class MainClient extends BaseRestClient {
       params,
     );
   }
-
-  /**
```

Comment:
This is not missing, I moved it up few lines, above the data stream. Makes more sense to be there I guess